src/home/homed-home-bus.c

```diff
@@ -201,10 +202,16 @@ int bus_home_method_unregister(
 
         assert(message);
 
+        const char *details[] = {
+                "uid", FORMAT_UID(h->uid),
+                "username", h->user_name,
+                NULL
+        };
+
         r = bus_verify_polkit_async(
                         message,
                         "org.freedesktop.home1.remove-home",
-                        /* details= */ NULL,
+                        details,
                         &h->manager->polkit_registry,
                         error);
```

Comment:
hmm, maybe add a new helper call home_verify_polkit_async() that takes a Home*, and fills in these thing on its own? could then also fill in the 4th argument on its own. 

that sounds much ncier than repeating the details here at so many places